src/@chakra-ui/gatsby-plugin/semanticTokens.ts

```diff
@@ -34,10 +34,10 @@ const semanticTokens = {
     primaryLight: { _light: "blue.100", _dark: "orange.100" },
     primaryDark: { _light: "blue.700", _dark: "orange.800" },
     primaryHover: { _light: "blue.300", _dark: "orange.300" },
-    body: { _light: "grey.700", _dark: "grey.100" },
-    bodyLight: { _light: "grey.500", _dark: "grey.100" },
-    disabled: { _light: "grey.400", _dark: "grey.500" },
-    background: { _light: "white", _dark: "grey.700" },
+    body: { _light: "gray.700", _dark: "gray.100" },
+    bodyLight: { _light: "gray.500", _dark: "gray.100" },
+    disabled: { _light: "gray.400", _dark: "gray.500" },
+    background: { _light: "white", _dark: "gray.700" },
     success: "green.500",
     error: "red.600",
```

Comment:
This has been changed to red 500 on the design system (red 600 was a typo)

Reply:
Good catch, ty